<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PET_brg_debug_page</class>
 <widget class="QWidget" name="PET_brg_debug_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>508</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="11" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>КТ1</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ВЧ2 (RF2)</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QComboBox" name="cb_tp1_sel"/>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Параметры синхронизации</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Период синхронизации, мс</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QComboBox" name="cb_rf1_sel"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Режим синхронизации</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QComboBox" name="cb_sync_mode"/>
     </item>
     <item row="8" column="0" colspan="3">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Контрольные точки</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QComboBox" name="cb_tp2_sel"/>
     </item>
     <item row="5" column="2">
      <widget class="QSpinBox" name="sb_sync_period"/>
     </item>
     <item row="10" column="2">
      <widget class="QComboBox" name="cb_rf2_sel"/>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>КТ2</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ВЧ1 (RF1)</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QComboBox" name="cb_log_severity"/>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Минимальный уровень важности сообщений</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Параметры протоколирования работы устройства</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btn_save_to_EEPROM">
     <property name="text">
      <string>Сохранить настройки в энергонезависимой памяти</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_set_default">
     <property name="text">
      <string>Установить настройки по умолчанию</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
